tests/stubs/cmsis_os.h

```diff
@@ -0,0 +1,8 @@
+#pragma once
```

Comment:
Do you think this stub is usable for other than unit-tests? You propose not to place it into the `tests/unit` directory subtree.

Reply:
It's the nearest `stubs` directory I've found. I could create `tests/unit/stubs`… on the other hand, once we have the fuzzers, they are not exactly unit tests and they would probably like to reuse it.

Reply:
ok, fair enough